client.go

```diff
@@ -160,13 +167,13 @@ func (c *client) init() error {
 	}
 
 	log.Debug("connect root region server...", c.rootServerName)
-	conn, err := newConnection(serverNameToAddr(c.rootServerName), false)
+	conn, err := newConnection(serverNameToAddr(c.rootServerName), ClientService)
```

Comment:
serverNameToAddr(c.rootServerName) is a thing
ClientService is anothor thing
they both are arguments of newConnection